tests/base.cpp

```diff
@@ -1,15 +1,15 @@
-#define CATCH_CONFIG_MAIN
-#include <catch2/catch.hpp>
-#include <MatrixCF/MatrixCF.hpp>
-
-
-TEST_CASE("Constructor"){
-    SECTION("default"){
-        mcf::Mat<int> A;
-
-        CHECK(A.getH() == 0);
-        CHECK(A.getW() == 0);
-        CHECK(A.getTotalSize() == 0);
-        CHECK(A.getArray() == nullptr);
-    }
+#define CATCH_CONFIG_MAIN
+#include "catch.hpp"
+#include <MatrixCF/MatrixCF.hpp>

```

Comment:
If you want we can change the `include path` for `MatrixCF` so that it is looks the same as in `README.md`